game/views.py

```
@@ -432,34 +446,43 @@ def offer_draw(request):
 
-    game = ChessGame.from_dict(game_data)
+        game = ChessGame.from_dict(game_data)
 
-    if game.mode == 'ai':
-        resigning_player = game.player_color
-    else:
-        resigning_player = game.current_turn
+        if game.mode == 'ai':
+            resigning_player = game.player_color
+        else:
+            resigning_player = game.current_turn
```

Ruff (0.15.12)

[warning] Use ternary operator `resigning_player = game.player_color if game.mode == 'ai' else game.current_turn` instead of `if`-`else`-block

Replace `if`-`else`-block with `resigning_player = game.player_color if game.mode == 'ai' else game.current_turn`

(SIM108)